docs/02-app/01-building-your-application/04-caching/index.mdx

```diff
@@ -0,0 +1,462 @@
+| [`const revalidate`](#segment-config-options)                       |              | Revalidate or Opt out | Revalidate or Opt out |             |
+| [`const dynamic`](#segment-config-options)                          |              | Cache or Opt out      | Cache or Opt out      |             |
+| [`cookies`](#cookies)                                               | Revalidate   | Opt out               |                       |             |
+| [`headers`, `useSearchParams`, `searchParams`](#dynamic-functions)  |              | Opt out               |                       |             |
+| [`React.cache`](#react-cache-function)                              |              |                       |                       | Cache       |
+| [`unstable_cache`](#unstable_cache) (Coming Soon)                   |              |                       |                       |             |
+
+### `<Link>`
+
+By default, the `<Link>` component automatically prefetches static routes (cached) and adds the React Payload to the Router Cache.
+
+To disable prefetching, you can set the `prefetch` prop to `false`. But this will not skip the cache permanently, the route segment will still be cached client-side when the user visits the route.
+
+Learn more about the [`<Link>` component](/docs/app/api-reference/components/link).
+
+### `router.prefetch`
+
+The `prefetch` option of the `useRouter` hook can be used to manually prefetch a route. This adds the React Payload to the Router cache.
+
+See the [`useRouter` hook](/docs/app/api-reference/functions/use-router) API reference.
+
+### `router.refresh`
+
+The `refresh` option of the `useRouter` hook can be used to manually refresh a route. This completely clears the Router Cache, and makes a new request to the server. `refresh` does not revalidate the Data or Render Cache, it reuses the cached data and rendering result, if it exists.
+
+The rendered result will be reconciled on the client without losing the React state or browser state (e.g. scroll position).
+
+See the [`useRouter` hook](/docs/app/api-reference/functions/use-router) API reference.
+
+### `fetch`
+
+Data returned from `fetch` is automatically cached.
+
+```jsx
+// Cached by default. `force-cache` is the default option and can be ommitted.
+fetch(`https://...`, { cache: 'force-cache' })
+```
+
+See the [`fetch` API Reference](/docs/app/api-reference/functions/fetch) for more options.
+
+### `fetch.options.cache`
+
+You can opt individual `fetch` requests out of data caching by setting the `cache` option to `no-store`:
+
+```jsx
+// Opt out of caching
+fetch(`https://...`, { cache: 'no-store' })
+```
+
+Since the render output depends on data, using `cache: 'no-store'` will also skip the Render Cache for the route where the `fetch` request is used. That is, the route will be dynamically rendered, but you can still have other cached data requests in the same route.
```

Comment:
```suggestion
Since the render output depends on data, using `cache: 'no-store'` will also skip the Full Route Cache for the route where the `fetch` request is used. That is, the route will be dynamically rendered, but you can still have other cached data requests in the same route.
```